### PR TITLE
fix: completions and gitlab merge

### PR DESCRIPTION
**Pull request by tk463**

### CODE REVIEW

File: bb8-utils-gitlab.env [hunk 1]
```diff
@@ -534,7 +534,8 @@ _t4dModBb8ListParse(){
                         [[ "$__REVIEW" == "" ]] && echo "    - ${plyellow}WAITING${pstd} $__NAME"
                     done
                 fi
-            elif [[ "$(echo "${__KEY}${__ID}${__REPOSITORY}${__STATUS}${__TITLE}" | grep "${_filtre}")" != "" ]] ; then 
+            #elif [[ "$(echo "${__KEY}${__ID}${__REPOSITORY}${__STATUS}${__TITLE}" | grep "${_filtre}")" != "" ]] ; then
```
Comment (QuentinNIO) [inline bb8-utils-gitlab.env]: cleanup ?

Comment (tk463) [reply bb8-utils-gitlab.env]: fixed, feature added back

File: bb8-utils-gitlab.env [hunk 2]
```diff
@@ -715,7 +722,7 @@ _t4dDebugInfoStart "_t4dModBb8UtilsGet" "$#" "$@"
 #
 ###
 #   - Example:
-#       _t4dModBb8UtilsGet https://<server>/<key>/<project>/-/merge_requests/<id>
+#       _t4dModBb8UtilsGet https://<server>/<key>/<project>/-/merge_requests/<id>>
```
Comment (QuentinNIO) [inline bb8-utils-gitlab.env:725]: humm this is strange.

**Review by QuentinNIO — APPROVED** [review]

File: bb8-utils-gitlab.env (hunk 2 above)
Comment (tk463) [reply bb8-utils-gitlab.env:725]: Added because it gets eaten in the generated md for some reason

Comment (QuentinNIO) [reply bb8-utils-gitlab.env:725]: humm yes hard to find a regex in md generator, but i don't know if adding a workaround like this is a good idea ^^

Comment (QuentinNIO) [reply bb8-utils-gitlab.env:725]: i think i fix this by using `{id}`. 

<> values are for usage purpose or highlights in terminal, basically surrounded by spaces not inside urls.

**Pull request merged**